src/transformers/models/maskformer/configuration_maskformer.py

```diff
@@ -105,7 +105,7 @@ def __init__(
         fpn_feature_size: int = 256,
         mask_feature_size: int = 256,
         no_object_weight: float = 0.1,
-        use_auxiliary_loss: bool = False,
+        auxiliary_loss: bool = False,
```

Comment:
Let's keep the original config value here. For two reasons: 
* Backwards compatibility - old configurations should be able to be loaded and produce the same result
* `use_auxiliary_loss` is a better name - `auxiliary_loss` sounds like the param is the loss itself

Reply:
I totally agree and modified as your suggestion!